atrium.ts

```diff
@@ -236,6 +261,11 @@ export class Account {
             "baseName": "last_payment",
             "type": "number"
         },
+        {
+            "name": "loanAmount",
```

Comment:
Just so I understand how this works: Is the idea here to resolve the different naming conventions between typescript and what the api returns?